modules/gapi/include/opencv2/gapi/imgproc.hpp

```diff
@@ -854,6 +926,107 @@ The algorithm normalizes the brightness and increases the contrast of the image.
  */
 GAPI_EXPORTS GMat equalizeHist(const GMat& src);
 
+//! @addtogroup gapi_shape
+//! @{
+/** @brief Finds contours in a binary image.
+
+The function retrieves contours from the binary image using the algorithm @cite Suzuki85 .
+The contours are a useful tool for shape analysis and object detection and recognition.
+See squares.cpp in the OpenCV sample directory.
+@note Since opencv 3.2 source image is not modified by this function.
```

Comment:
This note doesn't make sense for G-API module.

Reply:
I see, can agree with you. Input Mat actually never changes in G-API..
But on the one hand, I thought it could be important somehow to preserve the notes from OpenCV docs.
On the other hand - since it is a new operation for G-API, there is no need to mention the changes of API of OpenCV 3.2, right?
So, I should just remove this note, am I correct about the logic of this decision?

Reply:
removed